### PR TITLE
To do list _ Project1

### PR DESCRIPTION
**Pull request by diarisdiakite**
In this PR, I: 
- set up webpack
- added linters
- added and setup the html webpack plugin
- added a css file
- added a webpack dev-server
- set up a new project with webpack that is based on the webpack installation above-mentioned.
- created an **index.html** file and wrote my HTML markup. Create an empty **To Do List** 
placeholder (`<div>` or `<ul>` element). 
- set the **index.html** file set as a template using the *HTML Webpack Plugin* in the modules.exports of webpack.config.js
- Created an **index.js** file and set an array of some *to do tasks* with each task object containing the three keys: 
  1. description [`string`].
  2. completed [`bool`].
  3. index: [`number`].
- wrote a function to iterate over the tasks array and populated an HTML list item element for each task on page load render the dynamically created list of tasks in the dedicated placeholder. 
The list appear in order of the `index` values for each task.
- created a **style.css** and set rules for the **To Do List**. CSS is loaded by *Webpack Style/CSS Loader*.

### CODE REVIEW

**Review by didierganthier — COMMENTED** [review]
Hi @diarisdiakite ,

While you put in a lot of hard work in this project, unfortunately, I cannot proceed with the review of your code.



## Invalid Code Review Request 🚫

- You have submitted a project that doesn't have a README file. Consider adding a README following the [Microverse README template](https://github.com/microverseinc/readme-template).

Your Code Review Request will be marked as invalid in your Dashboard, but don't worry you can always try again! 💪

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

**Please note**
- To avoid confusion and delays, please do not open a new pull request for re-reviews. Use the same pull request submitted for the first review, unless otherwise instructed.

--------

_Invalid Code Review Request **does not count** towards the code reviews limit._

**Comment by diarisdiakite** [comment]
In this milestone, I added the README.md file

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  width: 5%;
+  background: url(images/circle.jpg) no-repeat;
+  background-size: 60%;
+}
+
+/* Form DIV */
+input {
+  width: 100%;
+  border: 0;
+}
```
Comment (didierganthier) [inline src/style.css:77]: [OPTIONAL] Well done overall 💯 but consider improving your styles according to [this video](https://youtu.be/AcUd-_Yjjqg)

**Review by didierganthier — APPROVED** [review]
Hi @diarisdiakite ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Setup Webpack ✔️ 
- Init tasks array ✔️ 
- Populated HTML ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

File: src/style.css (hunk 1 above)
Comment (diarisdiakite) [reply src/style.css:77]: Well received @didierganthier , thank you, I surely will.

**Pull request merged**